benchmark/parallel/egonets.jl

```diff
@@ -0,0 +1,63 @@
+using LightGraphs
```

Comment:
So, my main concern here is that we're not being forthcoming in what we're actually benchmarking. That is, the other benchmarks, if they fail, will tell us explicitly, "this function has a regression". I'm not sure how a failure here can be interpreted.